### PR TITLE
OPENTOK-31614 Add Sentry.io error monitoring

### PR DESCRIPTION
**Pull request by leahciMic**
#### What is this PR doing?

Add Sentry.io for error monitoring.

#### How should this be manually tested?

Clone this branch, add an error to the application, ensure it gets logged.

#### What are the relevant tickets?

[OPENTOK-31614](https://tokbox.atlassian.net/browse/OPENTOK-31614)

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -19,6 +19,10 @@
           ga('send', 'pageview');
 
         </script>
+        <script src="https://cdn.ravenjs.com/3.8.1/raven.min.js"></script>
+        <script type="text/javascript">
+          Raven.config('https://32febef98a3140ceb21ad35138178163@sentry.io/118721').install();
+        </script>
```
Comment (aiham) [inline views/index.ejs]: How come you've opted to use the CDN rather than install via packages and invoke `config` and `install` in each `app.js`?

Comment (leahciMic) [inline views/index.ejs]: We are happy to use CDNs for other resources in Meet, I followed suit :)

Comment (leahciMic) [inline views/index.ejs]: Sure thing!

**Comment by aiham** [comment]
@aullman We've somehow lost the API secret:

```
Error: Invalid arguments when initializing OpenTok: apiKey=44935341, apiSecret=undefined
```
https://travis-ci.org/opentok/opentok-meet/jobs/185153371

**Comment by aullman** [comment]
@aiham oh, that's another secret variable. We need to move this to be the same repo. I'll do that.

**Pull request closed without merging**